doc/recipe/preprocessor.rst

```diff
@@ -780,10 +780,6 @@ regridding is based on the horizontal grid of another cube (the reference
 grid). If the horizontal grids of a cube and its reference grid are sufficiently
 the same, regridding is automatically and silently skipped for performance reasons.
 
-The underlying regridding mechanism in ESMValCore uses
-:obj:`iris.cube.Cube.regrid`
-from Iris.
-
```

Comment:
any reason why you removing this, bud? Quite too general a statement indeed, but it's not wrong

Reply:
After this PR it will be wrong 😬 The new syntax is `regridded_cube = regridder(cube)`.

Reply:
aha - they changed the API indeed - maybe just drop a mention about the new func from iris? Up to you

Reply:
The old API is still there, but we are using a different one now that allows us to re-use the weights 😁 See https://scitools-iris.readthedocs.io/en/stable/userguide/interpolation_and_regridding.html#caching-a-regridder.

Reply:
yes, I forgot that from 2 hours ago when I reviewed - remember, am an old man :rofl: